Examples/CaseStudies/tvOSCaseStudies/FocusView.swift

```diff
@@ -48,7 +48,13 @@ let focusReducer = Reducer<FocusState, FocusAction, FocusEnvironment> {
             .padding()
 
           HStack(spacing: 40) {
-            ForEach(1..<11) { index in
+            ForEach(1..<6) { index in
+              Button(numbers[index]) {}
+                .prefersDefaultFocus(viewStore.currentFocus == index, in: self.namespace)
+            }
+          }
+          HStack(spacing: 40) {
+            ForEach(6..<11) { index in
```

Comment:
Unrelated fix to render the tvOS focus study more cleanly: 
<img width="958" alt="image" src="https://user-images.githubusercontent.com/658/146444137-2dd32364-385d-448d-af7c-09e4dba9f0a6.png">